cool-core/src/main/java/com/nus/cool/core/cohort/aggregator/UserCountAggregatorEvent.java

```diff
@@ -177,6 +174,60 @@ public void ageAggregateMetirc(BitSet ageOffset, InputVector timeVec, int birthD
 		}
 	}
 
+	public void ageAggregateMetirc(BitSet ageOffset, InputVector timeVec, int birthDay, int ageOff, int ageEnd, int ageInterval,
```

Comment:
shares large portion of codes with the codes above, should extract and reuse the common logic